client/extensions/woocommerce/app/dashboard/setup-tasks.js

```diff
@@ -108,6 +108,8 @@ class SetupTasks extends Component {
 			taxesAreSetUp,
 			translate
 		} = this.props;
+		const siteSlug = encodeURIComponent( '//' + site.slug );
+		const customizerUrl = getLink( 'https://:site/wp-admin/customize.php?wc-api-dev-tutorial=true&return=' + siteSlug, site );
```

Comment:
I know it's just a query arg that most users won't see or notice, but we should probably choose something different and not include `wc-api-dev` in the name. I don't have a great alternative idea but maybe `store-wpcom-nux` or something?

Reply:
works for me.